src/core/containers/ServerHtml.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component, PropTypes } from 'react';
+import ReactDOM from 'react-dom/server';
+import serialize from 'serialize-javascript';
+import Helmet from 'react-helmet';
+
+
+export default class ServerHtml extends Component {
+
+  static propTypes = {
+    appName: PropTypes.string.isRequired,
+    assets: PropTypes.object.isRequired,
+    component: PropTypes.object.isRequired,
+    env: PropTypes.string.isRequired,
+    sriData: PropTypes.object,
+    store: PropTypes.object.isRequired,
+  };
+
+  getStatic({filePath, type} = {}) {
+    const { env, sriData, appName } = this.props;
+    const leafName = filePath.split('/').pop();
+    const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
+    let sriProps = {};
+    // Only output files for the current app.
+    if (leafName.startsWith(appName)) {
```

Comment:
I like the idea of `includeSri`, I think we should conside moving the sri helper out in a future branch and/or as the need arises. Since we need to try and get back to adding core features ASAP.